yt_dlp/extractor/niconico.py

```diff
@@ -466,8 +532,8 @@ def get_video_info(*items, get_first=True, **kwargs):
             'duration': (
                 parse_duration(self._html_search_meta('video:duration', webpage, 'video duration', default=None))
                 or get_video_info('duration')),
-            'webpage_url': url_or_none(url) or f'https://www.nicovideo.jp/watch/{video_id}',
-            'subtitles': self.extract_subtitles(video_id, api_data),
+            'webpage_url': url_or_none(url) or f'https://www.nicovideo.jp/watch/{actual_video_id}',
+            'subtitles': self.extract_subtitles(actual_video_id, api_data, dmc_session_api_data),
```

Comment:
Wrong merge conflict resolution:

```suggestion
            'subtitles': self.extract_subtitles(actual_video_id, api_data),
```